pkg/pb/vouchers.proto

```diff
@@ -21,6 +21,19 @@ message Voucher {
 
 message VoucherRequest {}
 
+message VoucherResponse {
+    enum Status {
+        INVALID  = 0;
```

Comment:
do we ever use this invalid status? seems like it might not be needed

Reply:
Yeah, we can probably get rid of it

Reply:
it's nice to have the zero value default to "invalid"

Reply:
basically, if you construct this protobuf in go, all the values are initialized to zero values, and it's nice in many cases to have the status default to something besides something you opted in to